torch_dftd/functions/triplets_kernel.py

```diff
@@ -35,8 +35,8 @@ def _cupy2torch(array: cp.ndarray) -> Tensor:
 
 if _cupy_available:
     _calc_triplets_core_gpu_kernel = cp.ElementwiseKernel(
-        "raw int64 counts, raw int64 unique, raw int64 dst, raw T shift, raw int64 batch_edge, raw int64 counts_cumsum",
-        "raw int64 triplet_node_index, raw T multiplicity, raw T triplet_shift, raw int64 batch_triplets",
+        "raw int64 counts, raw int64 unique, raw int64 dst, raw int64 edge_indices, raw int64 batch_edge, raw int64 counts_cumsum",
+        "raw int64 triplet_node_index, raw T multiplicity, raw int64 edge_jk, raw int64 batch_triplets",
```

Comment:
The interface change also affects cupy implementation. `shift` to indices changes the data type.